packages/forma-36-react-components/src/components/IconButton/IconButton.css

```diff
@@ -7,7 +7,7 @@
   display: inline-block;
   cursor: pointer;
   border: 0;
-  padding: 0;
+  padding: 10px;
```

Comment:
Thanks a lot for the contribution @joneskj55 

Can you please use a spacing token here, maybe `spacing-xs`?
Usage would be similar to [Accordion.css](https://github.com/contentful/forma-36/blob/master/packages/forma-36-react-components/src/components/Accordion/Accordion.css#L26)

Reply:
Will do.

Reply:
I ended up using `var(--spacing-xs)` because it was closest in rem to the 10px of spacing. Please let me know if this will suffice. If not, I can use `xs` as you suggested.

Reply:
Great, thank you. will have a look, and test the component with its consumers.